src/Styleguide/Components/Pagination.tsx

```diff
@@ -68,22 +67,33 @@ export const LargePagination = (props: Props) => {
       )}
 
       <Box ml={4}>
-        <PrevButton onClick={() => props.onPrev()} />
+        <PrevButton
+          onClick={() => {
+            if (previous) {
```

Comment:
minor future QA item- if there is no previous page (or no next page), we probably want a greyed out/disabled state or something. Right now you get a cursor and it you click, but nothing happens.

Reply:
👍